Build_exclude.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ModuleInformation = @{
```

Comment:
This should definitely not be in here.
This is specific to your dev environment.

**Also, AtlassianPS uses `InvokeBuild`**

Reply:
That's fine, the gitignore should have stopped these files from going up there.

Reply:
you need to commit .gitignore with the changes before commit the files it is supposed to ignore.
you need to revert gitignore and commit the deletion of those files... then you can add it to gitignore again...

**however, please use this for stuff that is not part of the repo, but custom to your setup:**
https://www.peterbe.com/plog/gitinfoexclude

Reply:
Perfect, thank you for the link.